jaeger-spark-dependencies/src/main/java/io/jaegertracing/spark/dependencies/DependenciesSparkJob.java

```diff
@@ -39,24 +41,37 @@ public static void main(String[] args) throws UnsupportedEncodingException {
   }
 
   private static void run(String storage, LocalDate localDate) throws UnsupportedEncodingException {
+
     String jarPath = pathToUberJar();
     if ("elasticsearch".equalsIgnoreCase(storage)) {
       ElasticsearchDependenciesJob.builder()
           .jars(jarPath)
           .day(localDate)
+          .properties(prepareJobProperties())
           .build()
           .run();
     } else if ("cassandra".equalsIgnoreCase(storage)) {
       CassandraDependenciesJob.builder()
           .jars(jarPath)
           .day(localDate)
+          .properties(prepareJobProperties())
           .build()
           .run();
     } else {
       throw new IllegalArgumentException("Unsupported storage: " + storage);
     }
   }
 
+  static Map<String,String> prepareJobProperties(){
+    Map<String,String> properties = new HashMap<>();
+    // Key for naming external services as dependencies
+    String uninstrumentedKey = System.getenv("UNINSTRUMENTED_DEPENDENCIES_KEY");
```

Comment:
rename `UNINSTRUMENTED_DEPENDENCIES_KEY` to `PEER_SERVICE_TAG` and add it to readme.